### PR TITLE
Feature/tags filter

### PR DESCRIPTION
**Pull request by devahmedshendy**

### CODE REVIEW

File: Sources/SwiftUILogger/Internal/Extensions/File.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//
+//  File.swift
```
Comment (0xLeif) [inline Sources/SwiftUILogger/Internal/Extensions/File.swift]: File named File. 🙃

File: Sources/SwiftUILogger/Internal/SearchBar.swift [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+//  
+//
+//  Created by Ahmed Shendy on 12/11/2022.
+//
+
+import SwiftUI
+
+struct SearchBar: View {
+    
+    private var searchText: Binding<String>
```
Comment (0xLeif) [inline Sources/SwiftUILogger/Internal/SearchBar.swift]:
```suggestion
    @Binding private var searchText: String
```

File: Sources/SwiftUILogger/Internal/SearchBar.swift [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    private let placeholder: String
+    
+    init(searchText: Binding<String>, placeholder: String) {
+        self.searchText = searchText
```
Comment (0xLeif) [inline Sources/SwiftUILogger/Internal/SearchBar.swift]:
```suggestion
        self_.searchText = searchText
```

File: Sources/SwiftUILogger/Internal/SearchBar.swift [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    
+    var body: some View {
+        ZStack {
+            Rectangle()
+                .foregroundColor(Color(UIColor.lightGray).opacity(0.25))
+            
+            HStack(alignment: .center) {
+                Image(systemName: "magnifyingglass")
+                    .imageScale(.medium)
+                TextField(placeholder, text: searchText)
```
Comment (0xLeif) [inline Sources/SwiftUILogger/Internal/SearchBar.swift]:
```suggestion
                TextField(placeholder, text: $searchText)
```

**Review by 0xLeif — APPROVED** [review]
Small changes. Otherwise looks good.

File: Sources/SwiftUILogger/Internal/SearchBar.swift (hunk 3 above)
Comment (devahmedshendy) [reply Sources/SwiftUILogger/Internal/SearchBar.swift:16]: @0xLeif Oh this is new to me, thanks for these changes.

**Review by 0xLeif — APPROVED** [review]

**Review by 0xLeif — APPROVED** [review]

File: Sources/SwiftUILogger/Public/LoggerView.swift [hunk 5]
```diff
@@ -1,10 +1,31 @@
...
+        )
+        .sorted()
+    }
     
+    @State private var _filteredTags: OrderedSet<String> = []
+    private var filteredTags: OrderedSet<String> {
+        get {
+            _filteredTags.isEmpty ? OrderedSet(tags) : _filteredTags
+        }
+    }
```
Comment (0xLeif) [inline Sources/SwiftUILogger/Public/LoggerView.swift]: This seems to be extra code/ variables since tags is already an OrderedSet

**Review by 0xLeif — COMMENTED** [review]
Small thing

**Pull request merged**